### PR TITLE
Update to Babel 7

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
Update things to Babel 7.

## Testing done
Ran locally

## Acceptance criteria
- [x] Build passes

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,24 +80,33 @@
   },
   "homepage": "https://github.com/department-of-veterans-affairs/vets-website#readme",
   "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-decorators": "^7.0.0",
```
Comment (rianfowler) [inline package.json]: Didn't know we used these.

File: .babelrc [hunk 2]
```diff
@@ -15,27 +15,46 @@
          */
         "browsers": ["Chrome 61", "Firefox 60", "iOS 11", "Edge 16", "ChromeAndroid 67", "Safari 11"]
```
Comment (rianfowler) [inline .babelrc]: not a big deal but this doesn't match the comment above

**Review by rianfowler — APPROVED** [review]

File: .babelrc (hunk 2 above)
Comment (jbalboni) [reply .babelrc]: You mean the two browsers arrays? They're intentionally different.

File: package.json (hunk 1 above)
Comment (jbalboni) [reply package.json]: We don't. It got added automatically because it was in the stage2 preset, but I'll remove it.

File: .babelrc (hunk 2 above)
Comment (rianfowler) [reply .babelrc]: I see now thanks .

**Pull request merged**